controllers/authController.js

```diff
@@ -26,16 +28,35 @@ const refreshTokenExpirySecs = refreshTokenExpiryMillis / 1000;
 async function handleLogin(req, res) {
     if (!verifyBodyHasLoginFormData(req)) return; // If false, it will have already sent a response.
 
-    let { username, password } = req.body;
+    let { username, password, ip } = req.body;
```

Comment:
If they are the ones to send their IP, they can modify the code to change it every single time to get around this. Get their ip from `middleware/IP/getClientIP()`

Reply:
On it rn, didnt noticed this function sorry :)

Reply:
All good!